doc/src/build/sui-local-network.md

```diff
@@ -189,41 +189,57 @@ sui client gas
 
 ```
 
+## Pre-requisites for Building Apps locally 
+
+**Note:** To run the command you must have [pnpm](https://pnpm.io/installation) installed.
+
+```
+# install all the dependencies for the workspace first
+pnpm install
+pnpm turbo build
+```
+
+**Trouble shooting:** If turbo build is not working, check if you accidentally created a `package-lock.json`, and remove it first if exists.
+
 ## Set up a local Sui Explorer
 
 To connect the live Sui Explorer to your local network, open the URL:[https://explorer.sui.io/?network=local](https://explorer.sui.io/?network=local). The live version of Sui Explorer may not include recent updates added to the `main` branch of the Sui repo. To use Sui Explorer that includes the most recent updates, install and run Sui Explorer from your local clone of the Sui repo.
 
-**Note:** To run the command you must have [pnpm](https://pnpm.io/installation) installed.
-
 Run the following command from the `sui` root folder:
 
+**Note:** To run the command you must complete the `Pre-requisites for Building Apps locally` section first
```

Comment:
@randall-Mysten @ronny-mysten could you help me link to the section? I forgot the markdown syntax

Reply:
Can you help me to be on sui allowlisted in bybit? here's my email 
bybitemail: jubellgonzales182@gmail.com thank your mr. Randal🙏🙏🙏